6-anthos-install/README.md

```diff
@@ -188,6 +188,8 @@ With MCI, we need to select a cluster to be the configuration cluster. In this c
     kubectl --context ${CTX_INGRESS} create namespace istio-system
     ```
 
+    > **NOTE: Don't create another namespace with a different name if already exists "istio-system" created.**
```

Comment:
Given that MCI will be used to loadbalance between the istio-gateways in east and west clusters, we need to create istio-system namespace _(if it does not exist already)_ in Ingress cluster to establish namespace sameness.

Reply:
I think it really gets better. It's a simpler and more elegant way for the README.